### PR TITLE
AGU 2015 poster analysis

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

File: idea_tests/151218_AGU/munge/nldas.R [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Example NLDAS data for map figure
```
Comment (aappling-usgs) [inline idea_tests/151218_AGU/munge/nldas.R]: @jread-usgs - i promised to share how i used geoknife for this project. here it is.

File: idea_tests/151218_AGU/src/data_2_NLDAS.R [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Figure: Map of example NLDAS data
```
Comment (aappling-usgs) [inline idea_tests/151218_AGU/src/data_2_NLDAS.R]: @jread-usgs - here's how i used the data from geoknife to make the plot. `cache/nldas.RData` is created in `munge/nldas.R` as specified by the `makefile`.The US states map is not quite perfect - there's a little sliver of excess near Washington state - but it's close.

File: idea_tests/151218_AGU/makefile [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+cache/siteyears_clean.RData : munge/siteyears_clean.R munge/siteyears_lib.R cache/clean_2_GAM.RData
+	$(CALL_R) "--args outfile=$@" munge/siteyears_clean.R log/siteyears_clean.Rout
+
+cache/siteyears_GAM.RData : munge/siteyears_GAM.R munge/siteyears_lib.R cache/clean_2_GAM.RData
+	$(CALL_R) "--args outfile=$@" munge/siteyears_GAM.R log/siteyears_GAM.Rout
+
+cache/timing.RData : munge/timing.R munge/timing_lib.R cache/siteyears_GAM.RData
+	$(CALL_R) "--args outfile=$@" munge/timing.R log/timing.Rout
+
+cache/nldas.RData : munge/nldas.R
```
Comment (aappling-usgs) [inline idea_tests/151218_AGU/makefile]: @jread-usgs - here's the makefile piece of the geoknife application. also on line 54 of this file.

File: idea_tests/151218_AGU/munge/nldas.R (hunk 1 above)
Comment (jread-usgs) [inline idea_tests/151218_AGU/munge/nldas.R]: :+1: Thanks!

**Pull request merged**